docs/pages/configuration/images/image-definition/dockerfile.mdx

```diff
@@ -26,12 +26,12 @@ images:
     dockerfile: frontend/Dockerfile
     context: frontend/
 ```
-**Explanation:**  
+**Explanation:**
 - The first image would be built using the Dockerfile in `./Dockerfile` and the context path `./`.
 - The second image would be built using the Dockerfile in `./frontend/Dockerfile` and the context path `./frontend/`.
 - Switching the `context` for image `frontend` would assure that a statement like `ADD file.txt` or  `COPY file.txt .` in `./frontend/Dockerfile` would use the local file `./frontend/file.txt` instead of `./file.txt`.
```

Comment:
```suggestion
- Switching the `context` for image `frontend` assures that a statement like `ADD file.txt` or  `COPY file.txt .` in `./frontend/Dockerfile` uses the local file `./frontend/file.txt` instead of `./file.txt`.
```